homeworks/sviatoslav.taranenko_SviatoslavTaranenko/GamesProject/style.css

```diff
@@ -0,0 +1,19 @@
+.about__all .about__text .about__text_title {
```

Comment:
There is no reason to make so a long cascade. The last class would be enough.
I'll merge it, fix in your next task.